lib/taskana-core/src/main/java/pro/taskana/workbasket/internal/WorkbasketServiceImpl.java

```diff
@@ -75,7 +75,9 @@ public Workbasket getWorkbasket(String workbasketId)
         throw new WorkbasketNotFoundException(
             workbasketId, "Workbasket with id " + workbasketId + " was not found.");
       }
-      if (!taskanaEngine.getEngine().isUserInRole(TaskanaRole.ADMIN, TaskanaRole.BUSINESS_ADMIN)) {
+      if (!taskanaEngine
```

Comment:
Do we need this? Since this is not directly task related I would question this part